### 1>สร้าง function ชื่อ smartDict ซึ่ทำหน้าที่:
    - function รับบ input เป็น list 2 ตัวมีขนาดเท่ากัน โดย list แรกเป็น list ของตัวเลข list ที่สองเป็น list ของ text
    - ถ้าค่าใน list แรกเป็นเลขคี่ให้ค่าใน dict ที่เป็น output มีkey เป็นค่าจาก list ตัวที่1 และ value จาก list ตัวที่ 2
    - ถ้าค่าใน list แรกเป็นเลขคู่ให้ค่าใน dict ที่เป็น output มีkey เป็นค่าจาก list ตัวที่2 และ value จาก list ตัวที่ 1
    - output ของ function คือ dict ที่สร้างขึ้น

In [325]:
evennumber = [0,2,4,6,8,10]

#### ป.ล.เขียนการให้ index ใน listเป็นเงื่อนไขจำนวนคู่หรือคี่ไม่เป็นครับ

In [323]:

def smartDict(listnumber, listtext):
    if listnumber != evennumber:
        retdict = {}
        for index in range(len(listnumber)):
            retdict[listnumber[index]] = listtext[index]
    else:
        retdict = {}
        for index in range(len(listnumber)):
            retdict[listtext[index]] = listnumber[index]
    return retdict

In [315]:
smartDict([1,2,3],['apple','banana','orange'])

{1: 'apple', 2: 'banana', 3: 'orange'}

In [326]:
smartDict([7,1,4,6],['A','B','E','Zzz'])

{7: 'A', 1: 'B', 4: 'E', 6: 'Zzz'}

### 2>จากไฟล์ wuhan_GNome.txt จงเขียนโปรแกรมเพื่ออบคำถามต่อไปนี้
    - กรดอะมิโนใด (A, T, G, C)มีประมาณมากที่สุดและมีเท่าไร
    - จงหา sequence ที่มีขนาด 3 และ 4 ที่เกิดขึ้นซ้ำมากที่สุด

In [44]:
import pandas as pd 

In [45]:
DNAseq = pd.read_csv('C:\\Users\\ASUS\\Desktop\\pythonnew\\Wuhan_nCoV_GNome.txt',header = None)
DNAseq.head()

0
0  ATTAAAGGTTTATACCTTCCCAGGTAACAAACCAACCAACTTTCGA...
1  CGAACTTTAAAATCTGTGTGGCTGTCACTCGGCTGCATGCTTAGTG...
2  TAATTACTGTCGTTGACAGGACACGAGTAACTCGTCTATCTTCTGC...
3  TTGCAGCCGATCATCAGCACATCTAGGTTTCGTCCGGGTGTGACCG...
4  CCTGGTTTCAACGAGAAAACACACGTCCAACTCAGTTTGCCTGTTT...

In [47]:
def countChar(seq, character):
    charcount = 0
    for char in seq:
        if char == character:
            charcount = charcount + 1
    return charcount

In [49]:
ANum = 0
TNum = 0
GNum = 0
CNum = 0
for row in range(DNAseq.shape[0]):
    ANum += countChar(DNAseq.iloc[row][0],'A')
    TNum += countChar(DNAseq.iloc[row][0],'T')
    GNum += countChar(DNAseq.iloc[row][0],'G')
    CNum += countChar(DNAseq.iloc[row][0],'C')
    
print(f'there are {ANum} As, {TNum} Ts, {GNum} Gs, {CNum} Cs,')  

there are 8954 As, 9594 Ts, 5863 Gs, 5492 Cs,


### ตอบข้อ2.1 
### กรดอะมิโนที่มีมากที่สุดคือ T และมีจำนวน 9594 ตัว

In [56]:
def populate3charstring(seq):
    str_list = list()
    for i in range(len(seq)-2):
        str_list.append(seq[i:i+3])
    return str_list
   

In [57]:
populate3charstring('ATTAAAGG')

['ATT', 'TTA', 'TAA', 'AAA', 'AAG', 'AGG']

In [60]:
list_all_3char = list()
for row in range(DNAseq.shape[0]):
    list_all_3char = list_all_3char + populate3charstring(DNAseq.iloc[row][0])

In [61]:
len(list_all_3char)

29047

In [76]:
unique_3char = list(set(list_all_3char))
len(unique_3char)

64

In [77]:
unique_3char

['AAG',
 'GAA',
 'TGG',
 'GTA',
 'CAT',
 'GAG',
 'GTG',
 'GAC',
 'CAG',
 'ACT',
 'AGT',
 'CCA',
 'GTC',
 'TTT',
 'AGG',
 'CTG',
 'CTA',
 'CAA',
 'CTC',
 'ACG',
 'AAC',
 'AGA',
 'CCT',
 'GCT',
 'GGT',
 'TAT',
 'TCA',
 'TTA',
 'GAT',
 'ACA',
 'CCG',
 'CCC',
 'TAG',
 'ACC',
 'ATG',
 'CGC',
 'GGG',
 'AAT',
 'GCG',
 'CGG',
 'CAC',
 'TAC',
 'TCT',
 'CGT',
 'CGA',
 'TGA',
 'AGC',
 'TGT',
 'GGC',
 'GCA',
 'TCG',
 'ATA',
 'TGC',
 'AAA',
 'CTT',
 'ATT',
 'TTC',
 'TAA',
 'TCC',
 'ATC',
 'GGA',
 'GCC',
 'GTT',
 'TTG']

In [87]:
dict_3char = {}
for sub_str in unique_3char:
    num = 0
    for s in list_all_3char:
        if sub_str == s:
            num += 1
    dict_3char[sub_str] = num

In [85]:
dict_3char

{'AAG': 564,
 'GAA': 524,
 'TGG': 540,
 'GTA': 458,
 'CAT': 472,
 'GAG': 290,
 'GTG': 532,
 'GAC': 328,
 'CAG': 424,
 'ACT': 660,
 'AGT': 493,
 'CCA': 336,
 'GTC': 265,
 'TTT': 984,
 'AGG': 317,
 'CTG': 483,
 'CTA': 543,
 'CAA': 681,
 'CTC': 276,
 'ACG': 157,
 'AAC': 595,
 'AGA': 582,
 'CCT': 332,
 'GCT': 509,
 'GGT': 442,
 'TAT': 601,
 'TCA': 536,
 'TTA': 853,
 'GAT': 430,
 'ACA': 789,
 'CCG': 70,
 'CCC': 113,
 'TAG': 414,
 'ACC': 364,
 'ATG': 701,
 'CGC': 92,
 'GGG': 131,
 'AAT': 735,
 'GCG': 86,
 'CGG': 74,
 'CAC': 447,
 'TAC': 591,
 'TCT': 526,
 'CGT': 166,
 'CGA': 91,
 'TGA': 602,
 'AGC': 292,
 'TGT': 834,
 'GGC': 217,
 'GCA': 363,
 'TCG': 110,
 'ATA': 456,
 'TGC': 530,
 'AAA': 899,
 'CTT': 720,
 'ATT': 749,
 'TTC': 509,
 'TAA': 688,
 'TCC': 203,
 'ATC': 336,
 'GGA': 272,
 'GCC': 184,
 'GTT': 686,
 'TTG': 800}

In [88]:
maxVal = -100
maxKey = None
for k in dict_3char.keys():
    if dict_3char[k] > maxVal:
        maxVal = dict_3char[k]
        maxKey = k

In [93]:
maxKey

'TTT'

In [94]:
print(f'max 3-long-sequence is {maxKey} with frequency equal {maxVal}')

max 3-long-sequence is TTT with frequency equal 984


### ตอบข้อ 2.2.1 sequence ที่มีขนาด3และซ้ำกันมากที่สุด คือ TTT มีซ้ำจำนวน 984 

In [96]:
def populate4charstring(seq):
    str_list = list()
    for i in range(len(seq)-3):
        str_list.append(seq[i:i+4])
    return str_list
   

In [98]:
populate4charstring('ATTAAAGG')

['ATTA', 'TTAA', 'TAAA', 'AAAG', 'AAGG']

In [119]:
list_all_4char = list()
for row in range(DNAseq.shape[0]):
    list_all_4char = list_all_4char + populate4charstring(DNAseq.iloc[row][0])

In [120]:
len(list_all_4char)

28619

In [121]:
unique_4char = list(set(list_all_4char))
len(unique_4char)

256

In [122]:
dict_4char = {}
for sub_str in unique_4char:
    num = 0
    for s in list_all_4char:
        if sub_str == s:
            num += 1
    dict_4char[sub_str] = num

In [123]:
dict_4char

{'ATAC': 104,
 'CAGC': 69,
 'TCAT': 123,
 'ATGT': 202,
 'CATA': 106,
 'TCCA': 83,
 'AGGA': 87,
 'ATTG': 181,
 'CGGT': 29,
 'CGTG': 57,
 'AGTT': 171,
 'TAGT': 133,
 'GTGA': 121,
 'ATTT': 220,
 'CGAT': 31,
 'GTTT': 216,
 'AATT': 262,
 'CTCG': 32,
 'CAGT': 108,
 'GTGG': 109,
 'CAAA': 206,
 'GTCA': 96,
 'CCCG': 12,
 'CACC': 95,
 'ATAT': 107,
 'GAAG': 148,
 'CACG': 39,
 'GGCC': 30,
 'ATTC': 130,
 'TGCC': 92,
 'TCTA': 139,
 'AGAG': 129,
 'GTAT': 91,
 'TCGC': 23,
 'TGCA': 159,
 'GAGA': 77,
 'TCGG': 22,
 'GCAC': 93,
 'GACT': 93,
 'CGGA': 16,
 'ACAC': 162,
 'CTTT': 242,
 'CGTA': 39,
 'CGCC': 13,
 'ATGG': 168,
 'TATG': 184,
 'GGGA': 30,
 'ATCT': 124,
 'CTCC': 31,
 'CCAC': 78,
 'GCTT': 158,
 'TTTA': 277,
 'TGTA': 189,
 'GGTC': 28,
 'ATCA': 135,
 'CGCT': 27,
 'ACAG': 158,
 'TGCG': 31,
 'CATT': 152,
 'TGGC': 106,
 'ACCT': 135,
 'TACC': 105,
 'GCGG': 14,
 'CCGA': 16,
 'CCCA': 35,
 'AGAC': 113,
 'TATA': 132,
 'CATG': 118,
 'AACA': 245,
 'TGCT': 244,
 'AATG': 220,
 'CTAT': 161,
 'ACAA': 269,
 'GAGC': 

In [172]:
maxVal = 0
maxKey = None
for v in dict_4char.keys():
    if dict_4char[v] > maxVal:
        maxVal = dict_4char[v]
        maxKey = v

In [173]:
maxKey

'TGTT'

In [174]:
print(f'max 4-long-sequence is {maxKey} with frequency equal {maxVal}')

max 4-long-sequence is TGTT with frequency equal 318


### ตอบข้อ 2.2.2 sequence ที่มีขนาด4และซ้ำกันมากที่สุด คือ TGTT มีซ้ำจำนวน 318

### 3> จากไฟล์ data.xlsx สร้างตาราง
    - ใหม่ที่มี 2 columns โดย columns แรกชื่อว่า ภาษาอีสาน และมีคำ จาก columns ภาษาอีสานเดิมแต่เปลี่ยนคำที่เป๋น'/' เป็น','และ 
     columns ที่สองชื่อว่า ภาษาไทย และมีคำ จาก columns ภาษาอีสานเดิมแต่เปลี่ยนคำที่เป๋น'/' เป็น','เช่น
        จาก สวัสดี/สุผู้สุคน/เด้อ/จ้า สวัวสดี/ทุกๆคน /นะ/คะ
        เป็น สวัสดี,สุผู้สุคน,เด้อ,จ้า สวัวสดี,ทุกๆคน ,นะ,คะ
    - สร้างตาราง Dictionary โดยตารางประกอบด้วยสอง columns คือภาษาอีสานและ ภาษาไทย โดยคำใน column ภาษาอีสาน คือคำในภาษาอีสาน และ
    column ภาษาไทย คือคำแปลเป็นภาษากลางของภาษาอีสานใน column แรก (ใช้คำจากตาราง data.xlsx โดยคำที่อยู่ในลำดับเดียวกันของ columns
    ภาษาอีสาน และ ภาษาไทย คือคำแปลของกันและกันถ้าจำนวนของทั้งสอง coluomn เท่ากัน -ถ้าจำนวนคำระหว่างสอง columns ไม่เท่ากันให้ข้าม record นั้นไป) 

In [178]:
datatest = 'C:\\Users\\ASUS\\Desktop\\pythonnew\\data.xlsx'
data_df = pd.read_excel(datatest, sheet_name = 'v2', usecols = 'B:C',
                       skipfooter = 1)

In [179]:
data_df.head()

ภาษาอีสาน  \
0                  สะ/หวัด/ดี/จ้า/สุผู้สุคน/เด้อ/จ้า   
1  อ้อ/ได้/รับ/อั่น/ไปรษณีย์/เนาะ/จ้า/ไปรษณีย์/เด...   
2  เออ/เพิ่น/เขียน/อั่น/ลุง/อั่น/ลุง/เขียน/อั่น/อ...   
3  ลุง/ลุง/ฮา/หลาด/เป็น/เป็น/สา/มี/ของ/พี่/สาว/เน...   
4  โอ/เค้/ละ/กะ/มา/ต่อ/อั่น/เปิด/อั่น/นี้/เนาะ/จ่...   

                                             ภาษาไทย  
0                      สะ/หวัด/ดี/จ้า/ทุก ๆ คน/นะ/คะ  
1  อ้อ/ได้/รับ/เอิ่ม/ไปรษณีย์/นะ/คะ/ไปรษณีย์/เดี๋...  
2  เอ่อ/เขา/เขียน/เอิ่ม/ลุง/เอิ่ม/ลุง/เขียน/เอิ่ม...  
3  ลุง/ลุง/ฮา/หลาด/เป็น/เป็น/สา/มี/ของ/พี่/สาว/นะ...  
4  โอ/เค/แล้ว/ก็/มา/ต่อ/เอิ่ม/เปิด/อัน/นี้/นะ/คะ/...

In [182]:
colnames = ['ภาษาอีสาน' , 'ภาษาไทย']
Datatotal = pd.DataFrame.from_records([],columns = colnames)
for sn in range(32):
    df = pd.read_excel('C:\\Users\\ASUS\\Desktop\\pythonnew\\data.xlsx', sheet_name = sn, usecols = 'B:C', skipfooter = 1)
    Datatotal = pd.concat([Datatotal,df], axis = 0, ignore_index = True)

In [184]:
Datatotal.head()

ภาษาอีสาน  \
0                          สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1       กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2  เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4                     ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   

                                             ภาษาไทย  
0                              สวัสดี/ทุก ๆ คน/นะ/คะ  
1          กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์  
2  เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...  
3  นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...  
4                        ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน

In [186]:
Datatotal.shape

(975, 2)

In [191]:
Datatotal.iloc[3]['ภาษาอีสาน'].replace('/',',')

'นี่,จ้า,อา,หาร,เย็น,เป็น,ลา,ซาน,ย่า,กับ,ผัก,สด,เด้อ,จ้า'

In [194]:
New_colunm3 = list()
for n in range(Datatotal.shape[0]):
    New_colunm3.append(Datatotal.iloc[n]['ภาษาอีสาน'].replace('/',','))
New_colunm4 = list()
for n in range(Datatotal.shape[0]):
    New_colunm4.append(Datatotal.iloc[n]['ภาษาไทย'].replace('/',','))

In [195]:
Datatotal['ภาษาอีสาน(',')'] = New_colunm3
Datatotal['ภาษาไทย(',')'] = New_colunm4

In [196]:
Datatotal

ภาษาอีสาน  \
0                            สวัสดี/สุผู้สุคน/เด้อ/จ้า   
1         กิน/ข้าว/ก่อน/เด้อ/จ้า/ ไผ/สิ/ถ่า/เค้า/ดาวน์   
2    เค้า/ดาว/กะ/ขอ/ให้/เค้า/ดาว/อย่าง/มี/ความ/สุข/...   
3    นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4                       ขอ/ให้/เชิญ/แซ่บ/เด้อ/พี่/น้อง   
..                                                 ...   
970  นั่น/แหละ/คุย/กัน/ไป่/อูย/คุย/กัน/ไป่/เป็น/ปี/...   
971  กะ/เล๊ย/แบบ/ชี/วิต/ชวง/นั่น/แบบ/สิ/เอา/จัง/ได๋...   
972  เส็ด/แล่ว/กะ/เล๊ย/อ่า/เพิ่น/เซิน/ปะ/เออ/เซิน/ม...   
973  ซวง/นั่น/ข่อย/กะ/เขว/เนาะ/จ้า/สิ/ไป่/ตอ/หรือ/ส...   
974  ข่อย/กะ/เล๊ย/ตัด/สิน/ใจ/กั๊บ/มู/หว่า/เออ/ข่อย/...   

                                               ภาษาไทย  \
0                                สวัสดี/ทุก ๆ คน/นะ/คะ   
1            กิน/ข้าว/ก่อน/นะ/จ้ะ/ใคร/จะ/รอ/เค้า/ดาวน์   
2    เค้า/ดาวน์/ก็/ขอ/ให้/เค้า/ดาน์/อย่าง/มี/ความ/ส...   
3    นี่/จ้า/อา/หาร/เย็น/เป็น/ลา/ซาน/ย่า/กับ/ผัก/สด...   
4                          ขอ/ให้/เชิญ/อร่อย/นะ/ทุก/คน   
..                                                 ...   
970  นั้น/แหละ/คุย/กัน/ไป/อูย/คุย/กัน/ไป/เป็น/ปี/อย...   
971  ก็/เลย/แบบ/ชี/วิต/ช่วง/นั้น/แบบ/จะ/เอา/ยัง/ไง/...   
972  เสร็จ/แล้ว/ก็/เลย/อ่า/เขา/เชิญ/ปะ/เออ/เชิญ/มา/...   
973  ช่วง/นั้น/ฉัน/ก็/เคว้ง/อ่ะ/ค่ะ/จะ/ไป/ต่อ/หรือ/...   
974  ฉัน/ก็/เลย/ตัด/สิน/ใจ/กับ/เพื่อน/ว่า/เออ/ฉัน/จ...   

                                       (ภาษาอีสาน(, ))  \
0                            สวัสดี,สุผู้สุคน,เด้อ,จ้า   
1         กิน,ข้าว,ก่อน,เด้อ,จ้า, ไผ,สิ,ถ่า,เค้า,ดาวน์   
2    เค้า,ดาว,กะ,ขอ,ให้,เค้า,ดาว,อย่าง,มี,ความ,สุข,...   
3    นี่,จ้า,อา,หาร,เย็น,เป็น,ลา,ซาน,ย่า,กับ,ผัก,สด...   
4                       ขอ,ให้,เชิญ,แซ่บ,เด้อ,พี่,น้อง   
..                                                 ...   
970  นั่น,แหละ,คุย,กัน,ไป่,อูย,คุย,กัน,ไป่,เป็น,ปี,...   
971  กะ,เล๊ย,แบบ,ชี,วิต,ชวง,นั่น,แบบ,สิ,เอา,จัง,ได๋...   
972  เส็ด,แล่ว,กะ,เล๊ย,อ่า,เพิ่น,เซิน,ปะ,เออ,เซิน,ม...   
973  ซวง,นั่น,ข่อย,กะ,เขว,เนาะ,จ้า,สิ,ไป่,ตอ,หรือ,ส...   
974  ข่อย,กะ,เล๊ย,ตัด,สิน,ใจ,กั๊บ,มู,หว่า,เออ,ข่อย,...   

                                         (ภาษาไทย(, ))  
0                                สวัสดี,ทุก ๆ คน,นะ,คะ  
1            กิน,ข้าว,ก่อน,นะ,จ้ะ,ใคร,จะ,รอ,เค้า,ดาวน์  
2    เค้า,ดาวน์,ก็,ขอ,ให้,เค้า,ดาน์,อย่าง,มี,ความ,ส...  
3    นี่,จ้า,อา,หาร,เย็น,เป็น,ลา,ซาน,ย่า,กับ,ผัก,สด...  
4                          ขอ,ให้,เชิญ,อร่อย,นะ,ทุก,คน  
..                                                 ...  
970  นั้น,แหละ,คุย,กัน,ไป,อูย,คุย,กัน,ไป,เป็น,ปี,อย...  
971  ก็,เลย,แบบ,ชี,วิต,ช่วง,นั้น,แบบ,จะ,เอา,ยัง,ไง,...  
972  เสร็จ,แล้ว,ก็,เลย,อ่า,เขา,เชิญ,ปะ,เออ,เชิญ,มา,...  
973  ช่วง,นั้น,ฉัน,ก็,เคว้ง,อ่ะ,ค่ะ,จะ,ไป,ต่อ,หรือ,...  
974  ฉัน,ก็,เลย,ตัด,สิน,ใจ,กับ,เพื่อน,ว่า,เออ,ฉัน,จ...  

[975 rows x 4 columns]

In [198]:
Datatotal.iloc[0]['ภาษาอีสาน'].split('/')


['สวัสดี', 'สุผู้สุคน', 'เด้อ', 'จ้า']

In [199]:
Datatotal.iloc[0]['ภาษาไทย'].split('/')

['สวัสดี', 'ทุก ๆ คน', 'นะ', 'คะ']

In [202]:
list_dictionarytotal = []
for i in range(Datatotal.shape[0]) :
    a = Datatotal.iloc[i]['ภาษาอีสาน'].split('/')
    b = Datatotal.iloc[i]['ภาษาไทย'].split('/')
    if len(a) == len(b):
        X = len(a)
        for j in range(X):
            list_dictionarytotal.append(f'ภาษาอีสานคำว่า "{a[j]}" แปลว่า "{b[j]}" ในภาษาไทย')

In [206]:
list_dictionarytotal

['ภาษาอีสานคำว่า "สวัสดี" แปลว่า "สวัสดี" ในภาษาไทย',
 'ภาษาอีสานคำว่า "สุผู้สุคน" แปลว่า "ทุก ๆ คน" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เด้อ" แปลว่า "นะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "จ้า" แปลว่า "คะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "กิน" แปลว่า "กิน" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ข้าว" แปลว่า "ข้าว" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ก่อน" แปลว่า "ก่อน" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เด้อ" แปลว่า "นะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "จ้า" แปลว่า "จ้ะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า " ไผ" แปลว่า "ใคร" ในภาษาไทย',
 'ภาษาอีสานคำว่า "สิ" แปลว่า "จะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ถ่า" แปลว่า "รอ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เค้า" แปลว่า "เค้า" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ดาวน์" แปลว่า "ดาวน์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เค้า" แปลว่า "เค้า" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ดาว" แปลว่า "ดาวน์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "กะ" แปลว่า "ก็" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ขอ" แปลว่า "ขอ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ให้" แปลว่า "ให้" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เค้า" แปลว่า "เค้า" ในภาษาไทย',
 'ภาษาอีสานคำว่

In [207]:
len(list_dictionarytotal)

30852

In [208]:
dictionary_thai_isan = list(set(list_dictionarytotal))

In [209]:
dictionary_thai_isan

['ภาษาอีสานคำว่า "เอ้ย" แปลว่า "ขา" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ย่าน" แปลว่า "กลัว" ในภาษาไทย',
 'ภาษาอีสานคำว่า "สติก" แปลว่า "สติก" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เอ็กซเรย์" แปลว่า "เอ็กซเรย์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เซ้อ" แปลว่า "เซอร์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ปั้น" แปลว่า "ปั้น" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เบิ่ง" แปลว่า "ดู" ในภาษาไทย',
 'ภาษาอีสานคำว่า "คิด" แปลว่า "คิด" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ผู้" แปลว่า "คน" ในภาษาไทย',
 'ภาษาอีสานคำว่า "จันทร์" แปลว่า "จันทร์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "มึง" แปลว่า "มึง" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เมนท์" แปลว่า "เมนท์" ในภาษาไทย',
 'ภาษาอีสานคำว่า "หวาน" แปลว่า "หวาน" ในภาษาไทย',
 'ภาษาอีสานคำว่า "อุ่น" แปลว่า "อุ่น" ในภาษาไทย',
 'ภาษาอีสานคำว่า "พื่น" แปลว่า "พื้น" ในภาษาไทย',
 'ภาษาอีสานคำว่า "ธุระ" แปลว่า "ธุระ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "หยัง" แปลว่า "อะ" ในภาษาไทย',
 'ภาษาอีสานคำว่า "เคิล" แปลว่า "เคิล" ในภาษาไทย',
 'ภาษาอีสานคำว่า "แลม" แปลว่า "แรม" ในภาษาไทย',
 'ภาษาอีสานคำว่า "หึ" แปลว่า "หึ" ในภาษาไทย',
 